gulpfile.js/build.js

```diff
@@ -38,7 +38,7 @@ const RecentGuides = require('@lib/pipeline/recentGuides');
 // const roadmapImporter = require('@lib/pipeline/roadmapImporter');
 const gulpSass = require('gulp-sass');
 const lint = require('./lint.js');
-const importTasks = require('./import.js');
+const importTasks = require('./import/importWorkingGroups.js');
```

Comment:
The variable name doesn't make sense anymore since now it's only importing the WG importer. You could add a index.js file to the import directory that then exports all the import tasks.

Reply:
Not sure if I resolved this one correctly ..